apps/daimo-mobile/src/view/shared/SearchHeader.tsx

```diff
@@ -89,7 +93,14 @@ export function SearchHeader({
   return (
     <View style={styles.header}>
       <Animated.View key="back" style={backButton}>
-        <TouchableOpacity onPress={() => Keyboard.dismiss()} hitSlop={16}>
+        <TouchableOpacity
+          onPress={() => {
+            setPrefix(undefined);
+            Keyboard.dismiss();
+            ref.current?.closeInput();
```

Comment:
hmm im curious how this was working before (cancel keyboard > search prefix remains in place?)

Reply:
Blur keyboard always set the prefix to `undefined`
Focus set to `""`
And then typing until keyboard dismiss of keyboard then `undefined` right again